build/policheck_filetypes.xml

```diff
@@ -106,16 +106,6 @@
 <Ext Check="1" UseCustomParser="0" DllPath="" ClassName="">.xlt</Ext>
 </Extensions>
 </Type>
-<Type ID="8" Check="0">
-<Name>Pure Binary Files</Name>
-<Extensions>
-<Ext Check="0" UseCustomParser="0" DllPath="" ClassName="">.com</Ext>
-<Ext Check="0" UseCustomParser="0" DllPath="" ClassName="">.bin</Ext>
-<Ext Check="0" UseCustomParser="0" DllPath="" ClassName="">.tlb</Ext>
-<Ext Check="0" UseCustomParser="0" DllPath="" ClassName="">.drv</Ext>
-<Ext Check="0" UseCustomParser="0" DllPath="" ClassName="">.fon</Ext>
-</Extensions>
-</Type>
 <Type ID="9" Check="1">
```

Comment:
@bgavrilMS should I decrement the other IDs now that 8 and 29 are gone?

Reply:
I don't know, this file was generated by a tool. If our task passes, then use it.